### PR TITLE
Environmental impact

### PR DESCRIPTION
**Pull request by janschultecom**

### CODE REVIEW

File: environmental-impact.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Environmental impact
+---
+Let's face the truth: Conferences like LambdAle are not very environmental friendly. Tens or hundreds of people meeting for an event that typically lasts no more than a day or two. Some live close-by and come by public transport or car, but many fly in from other countries of the continent, or even travel from overseas. It is especially these flights that account for a lot of CO2 emissions. We don't believe in flat earth, but we do believe in global warming and that flying is a major contributor to it. So let's call a spade a spade - conferences are an environmental disaster.
```
Comment (cb372) [inline environmental-impact.md]: > many fly in from other countries of the continent, or even travel from overseas

This phrasing sounds a little unnatural. How about “many fly from another country or even another continent”?

File: README.md [hunk 2]
```diff
@@ -22,6 +23,17 @@ Then:
 
 4. Go to http://localhost:4000
 
+### Using docker
```
Comment (cb372) [inline README.md:26]: Nice

**Review by cb372 — COMMENTED** [review]
Looks great! Just one comment

File: environmental-impact.md (hunk 1 above)
Comment (janschultecom) [reply environmental-impact.md]: Done

**Pull request merged**